Google Cloud/google-drive-reports/ingest/parser.yml

```diff
@@ -10,11 +10,13 @@ stages:
   set_ecs_fields:
     actions:
       - set:
+          "@timestamp": "{{json_event.message.id.time}}"
           event.kind: "event"
           event.category: ["file"]
-          event.type: ["access"]
+          event.type: ["access", "edit"]
 
           user.id: "{{json_event.message.id.customerId}}"
+          user.email: "{{json_event.message.actor.email}}"
```

Comment:
I think the user != customer, so maybe we should create a custom field for this email. Also, we created a duplicate with l.25